microsoft-edge/webview2/reference/javascript/webviewmessageevent.yml

```
@@ -0,0 +1,44 @@
+### YamlMime:TSType
+name: WebViewMessageEvent
+uid: 'WebView2Script!WebViewMessageEvent:class'
+package: WebView2Script!
+fullName: WebViewMessageEvent
+summary: ''
+remarks: ''
+isPreview: false
+isDeprecated: false
+type: interface
+properties:
+  - name: additionalObjects
+    uid: 'WebView2Script!WebViewMessageEvent#additionalObjects:member'
+    package: WebView2Script!
+    fullName: additionalObjects
+    summary: >-
+      Additional DOM objects that are sent via PostJSONMessageWithAdditionalObjects. These objects can be of following

```

```suggestion
      Additional DOM objects that are sent via `PostJSONMessageWithAdditionalObjects`. These objects can be of the following
```
* the
* assuming backticks are supported here